WebDevStudios/create-block
# Sync with Block Starter repo

Closes #10, closes #9 

### DESCRIPTION ###
This PR updates the **template files** to sync with `WebDevStudios/wds-block-starter` repo. 

### SCREENSHOTS ###
The following screenshots are from generated plugin block starter after running the command.

**Plugin files**
![SS 1](https://user-images.githubusercontent.com/5747475/78353670-09271580-75dd-11ea-9aaa-0ea2b148a425.png)

**Plugin Dashboard**
![SS 2](https://user-images.githubusercontent.com/5747475/78353747-2b209800-75dd-11ea-9b38-f5536a61800f.png)

**Inside editor**
![SS 3](https://user-images.githubusercontent.com/5747475/78353843-599e7300-75dd-11ea-984b-a5c48789abc7.png)

**Frontend**
![SS 4](https://user-images.githubusercontent.com/5747475/78354037-b437cf00-75dd-11ea-8d65-68ae2ab90c6d.png)

### STEPS TO VERIFY ###
1. Clone this PR.
2. Inside the cloned directory run `npm install` then `npm run create-block WebDevStudios/TodoBlock`.
3. After running the command above, you'll have a new folder named `todo-block`. Copy that to a WordPress `wp-content/plugins`.
4. Test the generated plugin and confirm it works.
